os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -224,7 +224,7 @@ trble_result_e rtw_ble_client_connect(trble_conn_info* conn_info, bool is_secure
     conn_arg->remote_bd_type = conn_info->addr.type;
     conn_arg->conn_interval = conn_info->conn_interval;
     conn_arg->conn_latency = conn_info->slave_latency;
-    conn_arg->scan_timeout = conn_info->scan_timeout;
+    conn_arg->scan_timeout = conn_info->scan_timeout / 10;
```

Comment:
why divide 10 is necessary here?

Reply:
because the basic unit of RTK layer is 10 ms.
If BLE framework intend 1000 ms, it became 1000 * 10ms in RTK layer

Reply:
How about to add description to this line.

Reply:
How about to add description to this line.

Reply:
@pillip8282 
I added the comment about this